.golangci.yml

```diff
@@ -0,0 +1,149 @@
+  skip-dirs-use-default: true
+
+  # which files to skip: they will be analyzed, but issues from them
+  # won't be reported. Default value is empty list, but there is
+  # no need to include all autogenerated files, we confidently recognize
+  # autogenerated files. If it's not please let us know.
+  skip-files: []
+
+# output configuration options
+output:
+  # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
+  format: colored-line-number
+
+  # print lines of code with issue, default is true
+  print-issued-lines: true
+
+  # print linter name in the end of issue text, default is true
+  print-linter-name: true
+
+# all available settings of specific linters
+linters-settings:
+  ## Enabled linters:
+  govet:
+    # report about shadowed variables
+    check-shadowing: false
+    disable-all: false
+
+  golint:
+    min-confidence: 1.0
+
+  gocritic:
+    # Which checks should be enabled; can't be combined with 'disabled-checks';
+    # See https://go-critic.github.io/overview#checks-overview
+    # To check which checks are enabled run `GL_DEBUG=gocritic golangci-lint run`
+    # By default list of stable checks is used.
+    enabled-checks:
+      - argOrder # Diagnostic options
+      - badCond
+      - caseOrder
+      - dupArg
+      - dupBranchBody
+      - dupCase
+      - dupSubExpr
+      - nilValReturn
+      - offBy1
+      - weakCond
+      - boolExprSimplify # Style options here and below.
+      - builtinShadow
+      - emptyFallthrough
+      - hexLiteral
+      - ptrToRefParam
+      - underef
+      - equalFold
+
+linters:
+  disable-all: true
+  fast: false
+  enable:
+    - gocritic
+    - gofmt
+    - golint
+    - gosec
+    - govet
+    - misspell
+    - typecheck
+    - whitespace
+
+issues:
+  # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
+  max-issues-per-linter: 0
+
+  # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
+  max-same-issues: 0
+
+  # List of regexps of issue texts to exclude, empty list by default.
+  # But independently from this option we use default exclude patterns,
+  # it can be disabled by `exclude-use-default: false`. To list all
+  # excluded by default patterns execute `golangci-lint run --help`
+  exclude: ["composites"]
+
+  # Independently from option `exclude` we use default exclude patterns,
+  # it can be disabled by this option. To list all
+  # excluded by default patterns execute `golangci-lint run --help`.
+  # Default value for this option is true.
+  exclude-use-default: false
+
+  # Additional exclusion rules.
+  exclude-rules:
+    # Ignore gosec hardcoded credentials false positive.
+    - path: cmd/siac/
+      text: "G101: Potential hardcoded credentials"
+      linters:
+        - gosec
+    - path: cmd/siac/
+      text: "G107: Potential HTTP request made with variable url"
+      linters:
+        - gosec
+    - path: build/
```

Comment:
Same right? no build directory here yet.

Reply:
Yes, the entier`.golangci.yml` was copy-pasted.